### DISCUSSION TITLE
How to determine if a player is benched?

### DISCUSSION THREAD

**Discussion by JasonFreeberg** [cwendt94/espn-api#550] [ESPN API] [answered]
(Fantasy Football) 
Is there any way to determine if a Player on a Team's roster is benched? I can't seem to find a method or property to indicate that

**Comment by dtcarls** [comment]
In the [Box Player](https://github.com/cwendt94/espn-api/wiki/Box-Player-Class) class `slot_position` equal to 'BE' is benched. Typically I get the lineup of the box_score to get to the Box Player.

**Reply by cwendt94** [reply, marked as answer]
That is correct and a new field is being added to the player class `lineupSlot` which can be used to see which slot the player is in. Now in version v0.37.0
